TrackletAlgorithm/MatchProcessor.h

```diff
@@ -1259,13 +1224,13 @@ void MatchProcessor(BXType bx,
   //The next projection to read, the number of projections and flag if we have
   //more projections to read
 
-  ap_uint<8> vmstubsmask[16];
-#pragma HLS array_partition variable=vmstubsmask complete dim=1
+  //  ap_uint<8> vmstubsmask[16];
+  //#pragma HLS array_partition variable=vmstubsmask complete dim=1
 
- entriesloop:for(unsigned int i=0; i<16; i++) {
-#pragma HLS unroll
-    vmstubsmask[i]=instubdata.getBinMask8(bx,i);
-  }
+  // entriesloop:for(unsigned int i=0; i<16; i++) {
+  //#pragma HLS unroll
+  //  vmstubsmask[i]=instubdata.getBinMask8(bx,i);
+  //}
```

Comment:
@aryd I would prefer simply removing these lines:
```suggestion
```

Reply:
Removed (+ some other lines that were commented out)